cmd/kops/update_cluster.go

```diff
@@ -518,3 +537,38 @@ func completeLifecycleOverrides(cmd *cobra.Command, args []string, toComplete st
 	}
 	return completions, cobra.ShellCompDirectiveNoFileComp
 }
+
+func checkControlPlaneRunningVersion(ctx context.Context, clusterName string, version string) (string, error) {
+
+	configLoadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
```

Comment:
Nit:  I think we should try to share this "get kubeconfig" code with the rolling-update logic (I think that also connects to the cluster).  Not a blocker though.